[1;32mcommit 74804e377d4a54d1173d4393827d4e4b27e4d5d0[m
[1;33mdiff --cc libs/header_clean/header_clean.pl[m
[1;33mindex e8bcd92,5970580..ae279d0[m
[1;33m--- a/libs/header_clean/header_clean.pl[m
[1;33m+++ b/libs/header_clean/header_clean.pl[m
[1;35m@@@ -105,13 -104,21 +104,23 @@@[m [mfor (my $i = 0; $i <= $#input; $i++) [m
  	}[m
  }[m
  [m
[1;32m+ # Courtesy of github.com/git/git/blob/ab5d01a/git-add--interactive.perl#L798-L805[m
[1;32m+ sub parse_hunk_header {[m
[1;32m+     my ($line) = @_;[m
[1;32m+     my ($o_ofs, $o_cnt, $n_ofs, $n_cnt) =[m
[1;32m+         $line =~ /^@@ -(\d+)(?:,(\d+))? \+(\d+)(?:,(\d+))? @@/;[m
[1;32m+     $o_cnt = 1 unless defined $o_cnt;[m
[1;32m+     $n_cnt = 1 unless defined $n_cnt;[m
[1;32m+     return ($o_ofs, $o_cnt, $n_ofs, $n_cnt);[m
[1;32m+ }[m
[1;32m+ [m
  sub strip_empty_first_line {[m
[1;31m -	my $foo = shift(); # Array passed in by reference[m
[1;32m +	my $array = shift(); # Array passed in by reference[m
  [m
  	# If the first line is just whitespace remove it[m
[1;31m -	if (defined($foo->[0]) && $foo->[0] =~ /^\s*$/) {[m
[1;31m -		shift($foo);[m
[1;32m +	if (defined($array->[0]) && $array->[0] =~ /^\s*$/) {[m
[1;32m +		shift(@$array); # Throw away the first line[m
  	}[m
[1;32m +[m
[1;32m +	return 1;[m
  }[m
[1;33mdiff --git a/third_party/WebKit/Source/devtools/front_end/ui/Dialog.js b/third_party/WebKit/Source/devtools/front_end/ui/Dialog.js[m
[1;33mindex 4f9adf8..8c13743 100644[m
[1;33m--- a/third_party/WebKit/Source/devtools/front_end/ui/Dialog.js[m
[1;33m+++ b/third_party/WebKit/Source/devtools/front_end/ui/Dialog.js[m
[1;35m@@ -32,7 +32,7 @@[m
  * @constructor[m
  * @extends {WebInspector.Widget}[m
  */[m
[1;31m-WebInspector.Dialog = function()[m
[1;32m+[m[1;32mWebInspector.Dialog = function(isModalToElement)[m
 {[m
     WebInspector.Widget.call(this, true);[m
     this.markAsRoot();[m
[1;35m@@ -45,6 +45,10 @@[m [mWebInspector.Dialog = function()[m
 [m
     this._wrapsContent = false;[m
     this._dimmed = false;[m
[1;32m+[m[1;32m    this._isModalToElement = isModalToElement;[m
[1;32m+[m
[1;32m+[m[1;32m    this._glassPane = new WebInspector.GlassPane(relativeToElement, isModalToElement);[m
[1;32m+[m
     /** @type {!Map<!HTMLElement, number>} */[m
     this._tabIndexMap = new Map();[m
 }[m
[1;35m@@ -62,16 +65,16 @@[m [mWebInspector.Dialog.prototype = {[m
     /**[m
      * @override[m
      */[m
[1;31m-    show: function()[m
[1;32m+[m[1;32m    show: function(isModalToElement)[m
     {[m
         if (WebInspector.Dialog._instance)[m
             WebInspector.Dialog._instance.detach();[m
         WebInspector.Dialog._instance = this;[m
 [m
[1;31m-        var document = /** @type {!Document} */ (WebInspector.Dialog._modalHostView.element.ownerDocument);[m
[1;32m+[m[1;32m        var document = /** @type {!Document} */ (WebInspector.Dialog._modalHostView.element.ownerDocument, isModalToElement);[m
         this._disableTabIndexOnElements(document);[m
 [m
[1;31m-        this._glassPane = new WebInspector.GlassPane(document, this._dimmed);[m
[1;32m+[m[1;32m        this._glassPane = new WebInspector.GlassPane(document, isModalToElement);[m
         this._glassPane.element.addEventListener("click", this._onGlassPaneClick.bind(this), false);[m
         WebInspector.GlassPane.DefaultFocusedViewStack.push(this);[m
 [m
[33mcommit 6e50ba922f45d810b77de941a6a848df4a9ab44a[m[33m ([m[1;36mHEAD[m[33m -> [m[1;32mmaster[m[33m)[m
Author: Damien Robert <damien.olivier.robert+git@gmail.com>
Date:   Sun Mar 13 18:58:41 2016 +0100

    Rename symlink

[1mdiff --git a/d b/d[m
[1mdeleted file mode 100755[m
[1mindex 4bcfe98..0000000[m
[1m--- a/d[m
[1m+++ /dev/null[m
[36m@@ -1 +0,0 @@[m
[31m-d[m
[1mdiff --git a/d b/d[m
[1mnew file mode 120000[m
[1mindex 0000000..c59d9b6[m
[1m--- /dev/null[m
[1m+++ b/d[m
[36m@@ -0,0 +1 @@[m
[32m+[m[32md[m
\ No newline at end of file[m
[1mdiff --git a/e b/e[m
[1mdeleted file mode 120000[m
[1mindex c59d9b6..0000000[m
[1m--- a/e[m
[1m+++ /dev/null[m
[36m@@ -1 +0,0 @@[m
[31m-d[m
\ No newline at end of file[m

[33mcommit 8b9d73ca7c9e9da18bdb0beff3ab38c49b4e5ae1[m
Author: Damien Robert <damien.olivier.robert+git@gmail.com>
Date:   Sun Mar 13 18:58:31 2016 +0100

    symlink

[1mdiff --git a/e b/e[m
[1mnew file mode 120000[m
[1mindex 0000000..c59d9b6[m
[1m--- /dev/null[m
[1m+++ b/e[m
[36m@@ -0,0 +1 @@[m
[32m+[m[32md[m
\ No newline at end of file[m

[33mcommit 00c13fb19a63a7e753a2f06aaa214c05726c9bfb[m
Author: Damien Robert <damien.olivier.robert+git@gmail.com>
Date:   Sun Mar 13 18:58:21 2016 +0100

    copy + file permission

[1mdiff --git a/b2 b/c2[m
[1msimilarity index 100%[m
[1mcopy from b2[m
[1mcopy to c2[m
[1mdiff --git a/d b/d[m
[1mold mode 100644[m
[1mnew mode 100755[m

[33mcommit 8af4fb242d1040e291897a7d047da4dfbe97e9ef[m
Author: Damien Robert <damien.olivier.robert+git@gmail.com>
Date:   Sun Mar 13 18:58:02 2016 +0100

    Rename

[1mdiff --git a/b b/b2[m
[1msimilarity index 100%[m
[1mrename from b[m
[1mrename to b2[m

[33mcommit a1aad6c1153b923e0c5d14bc1bbbdf5e39d89591[m
Author: Damien Robert <damien.olivier.robert+git@gmail.com>
Date:   Sun Mar 13 18:57:39 2016 +0100

    Adding d, new line to b

[1mdiff --git a/b b/b[m
[1mindex 6178079..0939735 100644[m
[1m--- a/b[m
[1m+++ b/b[m
[36m@@ -1 +1,2 @@[m
 b[m
[32m+[m[32mnew line on b[m
[1mdiff --git a/d b/d[m
[1mnew file mode 100644[m
[1mindex 0000000..4bcfe98[m
[1m--- /dev/null[m
[1m+++ b/d[m
[36m@@ -0,0 +1 @@[m
[32m+[m[32md[m

[33mcommit fcf9b0274a1f034ced762c269f835e198693704a[m
Author: Damien Robert <damien.olivier.robert+git@gmail.com>
Date:   Sun Mar 13 18:57:39 2016 +0100

    Adding b

[1mdiff --git a/b b/b[m
[1mnew file mode 100644[m
[1mindex 0000000..6178079[m
[1m--- /dev/null[m
[1m+++ b/b[m
[36m@@ -0,0 +1 @@[m
[32m+[m[32mb[m

[33mcommit 6fa1c4ab2a8b2be408791d874155c95511ecc0ce[m
Author: Damien Robert <damien.olivier.robert+git@gmail.com>
Date:   Sun Mar 13 18:57:39 2016 +0100

    Adding a

[1mdiff --git a/a b/a[m
[1mnew file mode 100644[m
[1mindex 0000000..7898192[m
[1m--- /dev/null[m
[1m+++ b/a[m
[36m@@ -0,0 +1 @@[m
[32m+[m[32ma[m

Submodule ruby/gems/simplecolor contains untracked content
Submodule ruby/gems/simplecolor contains modified content
Submodule ruby/gems/simplecolor a5c060d..ac178d7:
  > Concatenate ANSI escape sequences inside the \e
  > Add possibility to copy colors
  > Sometimes \e[m is abused for \e[0m
  > More exceptions
  > Merge some color regexp and raise exception on bogus colors
  > Fix SimpleColor#uncolor!
  > Change ANSI Regexps
  > rake doc fails for now with Rake 11
  > Add uncolor tests
  > ColorWrapper: unify gestion of arg

Submodule script/others/republique f4c0876...0000000 (submodule deleted)
Submodule script/others/republique 0000000...f4c0876 (not checked out)
Submodule plam 0000000...c43a6e4 (new submodule)
Submodule plam c43a6e4...9215cb5 (commits not present)
Submodule plam 9215cb5..4b41ae0 (rewind):
  < Adding d, new line to b
Submodule plam de44db5...d6ed7c1:
  > Uhihih
  < Ohoho

[1mdiff --git i/a w/a[m
[1mindex 7898192..442406a 100644[m
[1m--- i/a[m
[1m+++ w/a[m
[36m@@ -1 +1,2 @@[m
 a[m
[7;34m+[m
[1mdiff --git i/b2 w/b2[m
[1mindex 0939735..d70291d 100644[m
[1m--- i/b2[m
[1m+++ w/b2[m
[36m@@ -1,2 +1,2 @@[m
[31m-b[m
[32m+[m[32mb[m[7;34m [m
 new line on b[m
[1mdiff --git i/b w/b[m
[1mindex 3cd10f0..3cc89da 100644[m
[1m--- i/b[m
[1m+++ w/b[m
[36m@@ -1,5 +1,5 @@[m
 b[m
 new line on b[m
 ohoh[m
[31m- [m
[32m+[m
 bl2[m

diff --git 1/2/umount 2/2/umount
new file mode 100755
index 0000000..bb9ce29
Binary files /dev/null and 2/2/umount differ
